documentation/managing_modules.md

```diff
@@ -0,0 +1,183 @@
+# 🧪 Managing modules in Bolt projects
+
+Bolt version 2.30.0 introduced module workflows for Bolt projects that improve
+dependency management and make it easier to add new modules to your Bolt
+project. These workflows, which include the `bolt module install`, `bolt module
+add`, and `bolt project migrate` commands, are experimental.
+
+## What's changed?
+
+- **Module installation directory:** Bolt now installs modules into the
+  `.modules/` directory instead of `modules/`. Avoid committing `.modules/` to
+  source control. Your users can download your Bolt project and use the `bolt
+  module install` command to download the required modules from the forge.
+- **Module configuration:** Your `bolt-project.yaml` file contains a `modules`
+  key that lists the direct module dependencies of your Bolt project. Unless you
+  need to pin a module to a specific version, avoid editing the `modules` list
+  directly. Instead, use the `bolt module add` command.
+- **Puppetfile:** The Puppet file is now a lock file. Bolt generates a
```

Comment:
```suggestion
- **Puppetfile:** The Puppetfile is now a [lock file](https://bundler.io/rationale.html#checking-your-code-into-version-control). Bolt generates a
```

Reply:
Or another link that explains what a lockfile is

Reply:
I'm going to add links to `Gemfile.lock` and `yarn.lock` below this